features/0023-did-exchange/README.md

```diff
@@ -321,8 +321,8 @@ The key used in the signed attachment must be verified against the invitation's
 #### Response Message Attributes
 
 * The `@type` attribute is a required string value that denotes that the received message is an exchange request.
-* The `~thread` block contains a `thid` reference to the `@id` of the request message.
-* The `did` attribute is a required string value and denotes DID in use by the responder. Note that this MAY NOT be the same DID used in the invitation.
+* The `~thread` decorator MUST be included. It contains a `thid` reference to the `@id` of the request message.
+* The `did` attribute MUST be included. It denotes the DID in use by the responder. Note that this MAY NOT be the same DID used in the invitation.
```

Comment:
Can we rephrase to remove the 'MAY NOT'? That's not actually RFC2119 verbiage.

Reply:
Discussed on today's B call:

The "MAY NOT" verbiage was already present in the RFC and not altered in any meaningful way by this PR. The purpose of this PR is to mark `did` as required.